components/event-publisher-proxy/pkg/sender/beb_test.go

```diff
@@ -72,8 +74,8 @@ func TestNewRequestWithTarget(t *testing.T) {
 	if req.Context() != ctx {
 		t.Errorf("HTTP request context does not match original context want: %#v, but got %#v", ctx, req.Context())
 	}
-	if got := req.Context().Value(ctxKey); got != ctxValue {
-		t.Errorf("HTTP request context key:value do not match mant: %v:%v but got %v:%v", ctxKey, ctxValue, ctxKey, got)
+	if got := req.Context().Value(ctxKey{}); got != ctxValue {
```

Comment:
required is a strong word in this context. but: https://pkg.go.dev/context#WithValue states that 
> The provided key must be comparable and should not be of type string or any other built-in type to avoid collisions between packages using context. 

a few lines prior to this change the ctxKey as now been defined as type struct{}.

This issue was found by the linter and instead of ignoring it (which imho would also be fine in this case) i decided to fix it properly.